.travis.yml

```diff
@@ -1,20 +1,22 @@
-# references:
-# * http://www.objc.io/issue-6/travis-ci.html
-# * https://github.com/supermarin/xcpretty#usage
-
+os: osx
+osx_image: xcode11.2
 language: objective-c
-osx_image: xcode9.4
 
-env:
-  - LC_CTYPE=en_US.UTF-8 LANG=en_US.UTF-8
+#coveralls
+before_install: rvm use $RVM_RUBY_VERSION
+install: 
+  - bundle install
+  - pod install
+after_success: slather
```

Comment:
```
12.33s $ slather
Slathering...
Slathered
```

😂 👏 👍

Reply:
slather is ruby gem :) so it's just for code coverage reporting